utils/src/treev2/quickPickWizard/GenericQuickPickStep.ts

```diff
@@ -64,16 +64,18 @@ export abstract class GenericQuickPickStep<TNode extends unknown, TContext exten
         }
     }
 
-    protected async getPicks(wizardContext: TContext): Promise<types.IAzureQuickPickItem<TNode>[]> {
-        const lastPickedItem: TNode | undefined = getLastNode(wizardContext);
+    protected async getPicks(wizardContext: TContext): Promise<types.IAzureQuickPickItem<unknown>[]> {
+        const lastPickedItem: unknown | undefined = getLastNode(wizardContext);
 
         // TODO: if `lastPickedItem` is an `AzExtParentTreeItem`, should we clear its cache?
-        const children = (await this.treeDataProvider.getChildren(lastPickedItem)) || [];
+        const childElements = (await this.treeDataProvider.getChildren(lastPickedItem)) || [];
+        const childItems = await Promise.all(childElements.map(async (childElement: unknown) => await this.treeDataProvider.getTreeItem(childElement)));
+        const childs: [unknown, vscode.TreeItem][] = childElements.map((childElement: unknown, i: number) => [childElement, childItems[i]]);

```

Comment:
Maybe a more accurate of name?
```suggestion
        const childPairs: [unknown, vscode.TreeItem][] = childElements.map((childElement: unknown, i: number) => [childElement, childItems[i]]);

```